App.js

```diff
@@ -60,34 +77,56 @@ const App = () => {
       }
 
       let userLocation = await Location.getCurrentPositionAsync({});
-      setLocation(userLocation);
+      setLocation([
+        userLocation.coords.latitude,
+        userLocation.coords.longitude,
+      ]);
     })();
   }, []);
   return (
-    <UserProvider
+    <VolunteerProvider
       value={{
-        user,
-        cart,
-        addToCart,
-        removeFromCart,
-        setNewUser,
-        location,
+        volunteer,
+        setVolunteer,
         installationId,
         setInstallationId,
-        submitOrder,
+        assignedLists,
+        setAssignedLists,
+        location,
+        setLocation,
+        allLists,
+        setAllLists,
       }}
     >
-      <NavigationContainer>
-        <RootStack.Navigator mode="modal">
-          <RootStack.Screen
-            name="Main"
-            component={MainStackScreen}
-            options={{ headerShown: false }}
-          />
-          <RootStack.Screen name="Your profile" component={LoginModal} />
-        </RootStack.Navigator>
-      </NavigationContainer>
-    </UserProvider>
+      <UserProvider
+        value={{
+          user,
+          cart,
+          addToCart,
+          removeFromCart,
+          setNewUser,
+          location,
+          installationId,
+          setInstallationId,
+          submitOrder,
```

Comment:
In my recent PR I created and added an `updateOrder` method so if a merge conflict arises we can add that in this file.